apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -214,6 +319,22 @@
           "properties": {
             "parameterKind": {
               "enum": ["flag"]
+            },
+            "addPhasesToCommand": {
+              "title": "Add phases to command",
+              "description": "Include the phases listed here in the command's execution if this parameter is specified.",
```

Comment:
Similar to below, adding phases to commands could suffer the same fate (a broken command).

For example, from my example repo, the command `rush push-notes --and-lint` would either not be able to lint anything, or it would need to exit with a failure because `lint` depends on `compile`, which isn't part of the command.

Reply:
(FWIW, since the ability to create "broken commands" isn't unique to these new properties, I don't think it's a stumbling block for them. But informing the user outright that the set of phases they are about to execute is not valid is probably the right call in all of these situations.)